ozaria/site/components/interactive/PageInteractive/BaseDraggableSlot.vue

```diff
@@ -0,0 +1,135 @@
+<script>
+  import draggable from 'vuedraggable'
+
+  export default {
+    components: {
+      'draggable': draggable
+    },
+
+    props: {
+      draggableGroup: {
+        type: String,
+        required: true
+      },
+
+      value: {
+        type: Object,
+        default: undefined
+      },
+
+      labelText: {
+        type: String,
+        default: ''
+      },
+
+      slotEmptyClass: {
+        type: String,
+        default: 'empty'
+      },
+
+      slotFilledClass: {
+        type: String,
+        default: 'filled'
+      }
+    },
+
+    data () {
+      return {
+        answers: (this.value) ? [ this.value ] : []
+      }
+    },
+
+    computed: {
+      draggableGroupConfig () {
+        return {
+          name: this.draggableGroup,
+          pull: true,
+          put: (this.answers.length === 0)
+        }
+      },
+
+      hasAnswer () {
+        return this.answers.length === 1
+      },
+
+      emptyFilledClass () {
+        return (this.hasAnswer) ? this.slotFilledClass : this.slotEmptyClass
+      }
+    },
+
+    methods: {
+      changed () {
+        this.$emit(
+          'input',
+          this.answers[0]
+        )
+      }
+    }
+  }
+</script>
+
+<template>
+  <div :class="[ 'slot-container', emptyFilledClass ]">
+    <div
+      class="expand slot-label"
+    >
+      {{ labelText }}
```

Comment:
What does labelText signify here? In other words, which text does it refer to from the interactives?

Reply:
The `labelText` is the empty space behind the draggable Label.
Screenshot where I hardcoded onto the labelText: 
<img width="688" alt="Screen Shot 2019-06-26 at 11 44 59 AM" src="https://user-images.githubusercontent.com/15080861/60206224-e2df7880-9807-11e9-8bbc-170f71888131.png">